app/components/SideBar/Logo/Logo.module.css

```diff
@@ -36,7 +36,9 @@
   background-image: var(--menu-mixer-icon);
   background-size: 16px 16px;
   cursor: pointer;
-  animation: spin 1s linear infinite;
+  /* XXX Need a different animation method due to performance of this
```

Comment:
Maybe a different `.gif` ? worth creating a separate issue I would say.

Reply:
Yah I'll leave this then call it out in an issue.  There's this and the pending indicators on transactions on the overview that I've noticed be huge perf hogs.